### ISSUE TITLE
Finetuning Learning Rate suggestions

### ISSUE THREAD

**Issue by initmaks** [CoinCheung/BiSeNet#145]
For fine-tuning, by default, the learning rate is set to the same schedule as with training from scratch, with a max value of lr=5e-2. I believe it should be smaller to no disturb the initial/learned weights too much. Was anyone able to find a good learning rate for finetuning BiSeNet-v2 with custom datasets? I have tested lr=1e-3, however, it might be too small the network doesn't improve even after 150k iterations.
I would appreciate any recommendations on the best hyper-parameter settings to finetune this repo's models.

**Comment by CoinCheung** [comment]
Hyper-parameters are determined case by case. Maybe you need to adjust your training profile with more experiments.
